exercises/forth/.meta/src/reference/kotlin/Forth.kt

```diff
@@ -0,0 +1,85 @@
+import java.util.*
+
+class Forth {
+
+    companion object {
+        private val NUMBER_PATTERN = Regex("""^-?\d+$""")
+    }
+
+    private var stack: Deque<Int> = ArrayDeque()
+    private var customOps: MutableMap<String, String> = mutableMapOf()
+    private var ops = mutableMapOf(
```

Comment:
Do you really need all these three fields declared as `var`s? There are no assignments for them, only referenced data structure mutations. They are effectively `val`s.

Reply:
Indeed, I changed them to vals.